src/cli/fuzzer_bridge/models.py

```diff
@@ -0,0 +1,140 @@
+"""Pydantic models for fuzzer output format v2."""
+
+from typing import Dict, List, Optional, Union
+
+from pydantic import BaseModel, Field, field_validator
+
+from ethereum_test_base_types import Address, Bytes, Hash, HexNumber
+
+# ruff: noqa: N815  # Allow mixedCase for JSON field compatibility
+
+
+class FuzzerAccount(BaseModel):
+    """Account definition in fuzzer output."""
+
+    balance: HexNumber
+    nonce: HexNumber = HexNumber(0)
+    code: Bytes = Bytes(b"")
+    storage: Dict[HexNumber, HexNumber] = Field(default_factory=dict)
+    privateKey: Optional[Bytes] = Field(None, alias="privateKey")
+
+    @field_validator("storage", mode="before")
+    @classmethod
+    def validate_storage(cls, v):
+        """Convert storage keys and values to HexNumber."""
+        if not v:
+            return {}
+        return {HexNumber(k): HexNumber(v_) for k, v_ in v.items()}
```

Comment:
This is basically `from ethereum_test_base_types import Account` so we can do:
```suggestion
class FuzzerAccount(Account):
    """Account definition in fuzzer output."""

    private_key: Hash | None = None
```